### ISSUE TITLE
Possible Bug: Cannot empty buckets from furnaces with any hopper

### ISSUE THREAD

**Issue by ValokAsura** [enjarai/OmniHopper#27]
Hello! Thank you for taking the time to make this cool mod! I discovered an issue with it today, however.

TL;DR: This mod (or possibly a dependency) prevents hoppers from removing empty buckets from furnaces. It causes vanilla hoppers to break as well.

When using lava to fuel a furnace, the empty bucket is being left in the fuel slot, and it can't being removed by a hopper placed under the furnace. This prevents automatic loading of fuel with hoppers. It doesn't matter if I use Omni-hoppers or Vanilla hoppers.

I have tons of mods enabled, so the first thing I did was install fresh instances of 1.21 Vanilla and 1.21 Fabric with no additional mods. The furnace/hopper combo worked as expected with both of those installs, removing empty buckets from the furnace so the next could be loaded with a hopper from side. Then I started the process of disabling mods from my highly-modded installation one at a time to find the culprit. Once I disabled only Omni-Hopper, my hoppers started removing the empty buckets.

I'm using Modrinth and I tried both versions that were published on June 26th. They have identical names but one says it's for 1.21 while the other 1.20.5, both have Changelogs that say "Updated to 1.21," and they have different file sizes. I also noticed that the "Supports: Fabric 1.21" version does not depend on Cooldown Coordinator. I'm not sure which version is actually ideal for 1.21, but both of them suffer the same issue for me. I tried another fresh 1.21 install with just Fabric, Cicada, and Omni-Hopper, but no luck.

Thank you for taking the time to read this! If I'm doing something stupid or can provide further info, please let me know.

![OmniHopper Versions](https://github.com/enjarai/OmniHopper/assets/40879862/9d32da19-7f69-4b76-aa41-012738515801)

![Enabled Mods](https://github.com/enjarai/OmniHopper/assets/40879862/efdd8356-2dae-414d-953d-15b322a09820)

![Bucket Stuck](https://github.com/enjarai/OmniHopper/assets/40879862/1c86dbff-7dbc-4c15-8855-eff5ea0f040f)

**Comment by Octol1ttle** [comment]
Seems like a duplicate of https://github.com/enjarai/OmniHopper/issues/23

**Comment by ValokAsura** [comment]
Sigh, I spent about an hour on this making sure I tested *everything* so I wouldn't waste your time with a useless bug report, but then I ended up doing it anyway. I only saw the open tickets here, not the closed ones. My apologies!

The gamerule, of course, did the trick, but I was unaware of its existence. Could you possibly add it to the mod description or put it in the configuration for the mod? Those were the first things I thought to check, as well as any possible config file. It might prevent you from having to deal with dummies like me in the future.

And lastly, do you have a suggestion regarding the same file names for possibly different versions? They both load and work fine for my 1.21 installation, but I wonder which one I should be using since they both say "Updated to 1.21" and have different dependencies and version support listed.
